app/Jobs/ProcessPDFDocument.php

```diff
@@ -2,26 +2,31 @@
 
 namespace App\Jobs;
 
+use App\wait_process;
 use Illuminate\Bus\Queueable;
 use Illuminate\Queue\SerializesModels;
 use Illuminate\Queue\InteractsWithQueue;
 use Illuminate\Contracts\Queue\ShouldQueue;
 use Illuminate\Foundation\Bus\Dispatchable;
+use Mockery\Exception;
```

Comment:
I wouldn't trust your IDE on that one.

http://docs.mockery.io/en/latest/

Reply:
Thank's, removed in last commit